README.md

```diff
@@ -14,21 +14,98 @@ It is work in progress, but functions well.
 | ![](before.png)                                |                                 ![](after.png) |
 
 # Supported Languages
-- [x] [C][tree-sitter-c]
-- [x] [C++][tree-sitter-cpp]
-- [ ] CSS
-- [x] [D][tree-sitter-d]
-- [x] [Diff][tree-sitter-diff]
-- [x] [Go][tree-sitter-go]
-- [x] [go.mod][tree-sitter-go-mod]
-- [ ] HTML
-- [x] [Javascript/JSX][tree-sitter-javascript]
-- [x] [Julia][tree-sitter-julia]
-- [x] [Lua][tree-sitter-lua]
-- [x] [Rust][tree-sitter-rust]
-- [x] [Zig][tree-sitter-zig]
-
-If you want more languages supported, open an issue.
+
+Any language supported by `tree-sitter` should (in theory) be supported as well.
+
+It is possible to use grammar installed locally or directly from their git repository. Additionally 
+it is possible to specify where to find the queries of the language as well as use sub path for projects
+containing more then one language.
+## Default grammars
+
+A few grammars have default configurations, pre-compiled parsers and custom queries:
+
+The supported languages are:
+ 
+- [C][tree-sitter-c]
+- [C++][tree-sitter-cpp]
+- [D][tree-sitter-d]
+- [Diff][tree-sitter-diff]
+- [Go][tree-sitter-go]
+- [go.mod][tree-sitter-go-mod]
+- [Javascript/JSX][tree-sitter-javascript]
+- [Julia][tree-sitter-julia]
+- [Lua][tree-sitter-lua]
+- [Rust][tree-sitter-rust]
+- [Zig][tree-sitter-zig]
+
+To use any of this grammars simply add this to your conifg 
+
+```lua
+local egconfig = require "plugins.evergreen.config"
+
+egconfig.addGrammar {
+  lang = "LANGUAGE",
+  precompiled = true -- optional: default true, if false use default git repository to install grammar
+}
+```
```

Comment:
change explanation of precompiled option as seen fit, but this is what i would write

```suggestion
```lua
local config = require "plugins.evergreen.config"

config.addGrammar {
  lang = "LANGUAGE",
  precompiled = true -- defaults to true, specifies if language parser is precompiled and available on the evergreen builds repo. all supported languages have precompiled parsers.
}
```
```